app/views/layouts/application.html.erb

```diff
@@ -10,6 +10,9 @@
   </head>
 
   <body>
+    <%= flash[:alert] %>
```

Comment:
That reminds me, I should add a flash to the item update in the merchant invoice show page -BF